src/main/app.ts

```diff
@@ -16,30 +29,30 @@ const nunjucksEnv = nunjucks.configure([path.join(__dirname, "views"),
     express: app
 });
 
-nunjucksEnv.addGlobal("cdnUrlCss", process.env.CDN_URL_CSS);
-nunjucksEnv.addGlobal("cdnUrlJs", process.env.CDN_URL_JS);
-nunjucksEnv.addGlobal("cdnHost", process.env.CDN_HOST);
-nunjucksEnv.addGlobal("chsUrl", process.env.CHS_URL);
+app.set("views", path.join(__dirname, "views"));
+app.set("view engine", "njk");
+nunjucksEnv.addGlobal("cdnUrlCss", CDN_URL_CSS);
+nunjucksEnv.addGlobal("cdnUrlJs", CDN_URL_JS);
+nunjucksEnv.addGlobal("cdnHost", CDN_HOST);
+nunjucksEnv.addGlobal("chsUrl", CHS_URL);
+nunjucksEnv.addGlobal("SERVICE_NAME", APPLICATION_NAME);
 
 app.enable("trust proxy");
 
-declare module "express-session" {
-    export interface SessionData {
-      user: { [key: string]: any };
-    }
-  }
-app.use(session({
-    secret: "123456",
-    resave: false,
-    saveUninitialized: true
-}));
+// declare module "express-session" {
```

Comment:
Commented code can be removed, if not required

Reply:
Removed.